setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
We absolutely odn't want to catch random errors, if there's a `NameError` for example, that indicates a bug, and we don't want to silence this.